app/controllers/EightBallController.scala

```diff
@@ -0,0 +1,30 @@
+package controllers
+
+import javax.inject._
+
+import play.api.mvc._
+
+import models.EightBallReply
+
+@Singleton
+class EightBallController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
+
+  def randomReply = Action {
+    val rndString = EightBallReply.getRndBall.content
+    Ok(views.txt.text(rndString))
+  }
+
+  def getMessages = Action {
+    val allMsgs = (EightBallReply.getAll map ( x => x.content ) ).
+      fold("")((x:String, y:String) => x + "\n" + y)
+    Ok(views.txt.text(allMsgs))
+  }
+
+  def getMessage(id: Int) = Action {
+    EightBallReply.get(id) map { x =>
+      Ok(views.txt.text(x.content))
+    } getOrElse {
+      Ok(views.txt.text("Nothing to see here!"))
```

Comment:
Can you change this from `Ok` to `NotFound`, please? It will show the same in the browser, but the status will be 404 instead of 200.